utils.py

```diff
@@ -219,12 +220,15 @@ def add_opacity(rgb, opacity):
     rgba = (first + "," + str(opacity) + ")").replace("rgb", "rgba")
     return rgba
 
+
 def get_prefixed_path(pathname):
     base_path = get_app().config.requests_pathname_prefix
     return f"{base_path.rstrip('/')}/{pathname}"
 
+
 def get_login_path():
-    return get_prefixed_path('login')
+    return get_prefixed_path("login")
```

Comment:
@bhupatiraju please remove the rule in your auto-formatter to replace single/double quotes. It introduces unnecessary noise for code review.